e2e/playwright.config.js

```diff
@@ -21,7 +21,10 @@ export default defineConfig({
   // Opt out of parallel tests on CI.
   workers: process.env.CI ? 1 : undefined,
   // Use 'blob' for CI to allow merging of reports. See https://playwright.dev/docs/test-reporters
-  reporter: process.env.CI ? 'blob' : 'html',
+  reporter: process.env.CI
+  ? [['blob']]  :
+    // Never open the HTML report to prevent terminal from hanging on test failure
```

Comment:
Nit: I think the comment isn't accurate if running natively

Reply:
Maybe something like "Don't open the HTML report since it hangs when running inside a container. Use make e2e-show-report for opening the HTML report"